worlds/lingo/docs/en_Lingo.md

```diff
@@ -0,0 +1,60 @@
+# Lingo
+
+## Where is the settings page?
+
+The [player settings page for this game](../player-settings) contains all the options you need to configure and export a
+config file.
+
+## What does randomization do to this game?
+
+There are a couple of modes of randomization currently available, and you can pick and choose which ones you would like
+to use.
+
+* **Door shuffle**: There are many doors in the game, which are opened by completing a set of panels. With door shuffle
+  on, the doors become items and only open up once you receive the corresponding item. The panel sets that would
+  ordinarily open the doors become locations.
+
+* **Color shuffle**: There are ten different colours of puzzle in the game, each representing a different mechanic. With
+  color
+  shuffle on, you would start with only access to white puzzles. Puzzles of other colours will require you to receive an
+  item in order to solve them (e.g. you can't solve any red puzzles until you receive the "Red" item).
+
+* **Panel shuffle**: Panel shuffling replaces the puzzles on each panel with different ones. So far, the only mode of
+  panel shuffling is "rearrange" mode, which simply shuffles the already-existing puzzles from the base game onto
+  different panels.
+
+* **Painting shuffle**: This randomizes the appearance of the paintings in the game, as well as which of them are warps,
+  and the locations that they warp you to. It is the equivalent of an entrance randomizer in another game.
+
+## What is a "check" in this game?
+
+Most panels / panel sets that open a door are now location checks, even if door shuffle is not enabled. Various other
+puzzles are also location checks, including the achievement panels for each area.
+
+## What about wall snipes?
+
+"Wall sniping" refers to the fact that you are able to solve puzzles on the other side of opaque walls. This feature is
+not affected by randomization, but you are almost never required to perform a wall snipe. The exceptions to this are:
```

Comment:
Wording might be a bit confusing here? I dunno...

I might go with "This randomiser does not change how wall snipes work, and will generally not require the use of wall snipes. However some exceptions to this are:"

Reply:
I tweaked it using your suggestion. I'm also considering just modding out the three required wall snipes, which would render the list of exceptions unnecessary.